comp/metadata/inventoryagent/README.md

```diff
@@ -62,6 +62,9 @@ The payload is a JSON dict with the following fields
     `network_config.enabled` config option in `system-probe.yaml`).
   - `feature_oom_kill_enabled` - **bool**: True if the OOM Kill check is enabled for System Probe (see: `system_probe_config.enable_oom_kill` config option in `system-probe.yaml`).
   - `feature_tcp_queue_length_enabled` - **bool**: True if TCP Queue Length check is enabled in System Probe (see: `system_probe_config.enable_tcp_queue_length` config option in `system-probe.yaml`).
+  - `python` - **object**:
```

Comment:
What about `python_environment` ? Also, this will play poorly in REDAPL as each field in this payload match a column in the DB. So a nested dict will require manual post processing in the backend.